### ISSUE TITLE
iOS 13.4.1 extract the AppleMediaServices framework failed

### ISSUE THREAD

**Issue by jiaxw32** [arandomdev/DyldExtractor#11]
When I try to extract the AppleMediaServices framework, an error happened.  The log as follows:

```
python3 extractor.py -f AppleMediaServices -o AppleMediaServices dyld_shared_cache_arm64
Extracting AppleMediaServices
17:12:47:659 StubConverter.py [WARNING ] : No symbol for: 0x18d41b694
17:12:47:705 StubConverter.py [WARNING ] : No symbol for: 0x1902f2ca0
17:12:47:775 StubConverter.py [WARNING ] : No symbol for: 0x190300688
17:12:47:788 StubConverter.py [WARNING ] : No symbol for: 0x18aa6fb24
17:12:47:810 StubConverter.py [WARNING ] : No symbol for: 0x18794bdd8
17:12:47:816 StubConverter.py [WARNING ] : No symbol for: 0x18d43f5e8
17:12:47:825 StubConverter.py [WARNING ] : No symbol for: 0x18d44a8cc
17:12:47:857 StubConverter.py [WARNING ] : No symbol for: 0x1903217d4
Traceback (most recent call last):
  File "extractor.py", line 146, in <module>
    extractImage(dyldFile, dyldFile.images[targetImageData[0]], args.output)
  File "extractor.py", line 70, in extractImage
    Converter.StubConverter(machoFile, dyld).convert()
  File "~/Workspace/OpenSource/DyldExtractor/DyldExtractor/Converter/StubConverter.py", line 113, in convert
    self.fixCallsites()
  File "~/Workspace/OpenSource/DyldExtractor/DyldExtractor/Converter/StubConverter.py", line 436, in fixCallsites
    targetFunc = self.resolveTarget(target)
  File "~/Workspace/OpenSource/DyldExtractor/DyldExtractor/Converter/StubConverter.py", line 279, in resolveTarget
    sect = image.segmentForAddr(currentAddr)[1]
TypeError: 'NoneType' object is not subscriptable
```

1. iOS version: 13.4.1
2. Device: iPhone 7
3. Terminal command: `python3 extractor.py -f AppleMediaServices -o AppleMediaServices dyld_shared_cache_arm64`

**Comment by arandomdev** [comment]
Try this...

In DyldExtractor/Converter/StubConverter.py, on line 279 change
```
sect = image.segmentForAddr(currentAddr)[1]
```
to
```python
# null guard.
sect = image.segmentForAddr(currentAddr)
if not sect:
    return -1
else:
    sect = sect[1]
```

If this gets rid of the exception I'll do a quick commit soon.

**Comment by jiaxw32** [comment]
@arandomdev works fine now, thanks for your work!

**Comment by arandomdev** [comment]
fixed with #778b8c3ffa294cba7418861098e6a8b57c0f65a4
